### DISCUSSION TITLE
[Projects beta] Show on Issues Filters

### DISCUSSION THREAD

**Discussion by mimisabel** [community/community#10876] [Projects and Issues]
It would be nice to be able to filter Issues that are associated to one of the [Projects beta] on Issues Filters

![Group 2](https://user-images.githubusercontent.com/68071180/152044983-a4d6e563-f095-4316-b7a4-d0c9b139c5db.png)

**Comment by kimchi-jjigae** [comment]
Agreed.

I also need to find the ones that are not in any project so that I can add them to that project. The `no:project` does not work.

And this has been requested many times, e.g.
#10749, #11148, #4448, #6332,  #9535, #9606

And along the same lines #6765.

This looks promising though to help with that workflow https://github.com/github/roadmap/issues/286
